articlequality/extractors/extractor.py

```diff
@@ -131,6 +135,17 @@ def extract(self, page, verbose=False):
             for ob in first_observations.values():
                 yield ob
 
+    def invert_reverted_status(self, reverteds, revisions, labelings):
+        for rev_id in reverteds:
```

Comment:
If a reverted edit is reverted again, that doesn't mean it was restored.  I think it needs to be "reverted_to" or some revision that occurred after it was "reverted_to".